dapp-core/src/test/resources/last_block.txt

```diff
@@ -0,0 +1 @@
+3
```

Comment:
0

Reply:
I have refactored this part a little bit